plugins/cpp/test/src/cpppropertiesservicetest.cpp

```diff
@@ -35,7 +37,9 @@ class CppPropertiesServiceTest : public ::testing::Test
   }
 
   void checkProperties(model::FileId fid_, int line_, int col_,
-    std::map<std::string, std::string>& expectedLines_)
+    std::map<std::string, std::string>& expectedValues_,
+    std::vector<std::string> ignoredValues_ = {},
+    std::vector<std::string> optionalProperties_ = {})
```

Comment:
Input variables should be passed by const reference.

Reply:
That's not true. This works for me:
```.cpp
void f(const std::vector<int>& v = {}) {}
int main() { f({}); }
```
Const reference keeps these temporaries alive.